### PR TITLE
[PRO-2986] Use kafka instead of nats in docker-compose.yml

### PR DESCRIPTION
**Pull request by jerrytrieu**

### CODE REVIEW

**Comment by jerrytrieu** [comment]
@koshelev should we still use the `advancedtelematic/confluent-kafka` image?

**Comment by koshelev** [comment]
@jerrytrieu No. We are not using this image anymore
```sh
  docker run -d -p 2181:2181 --name zookeeper wurstmeister/zookeeper

  docker run -d -p 9092:9092  \
    --link zookeeper \
    -e KAFKA_ZOOKEEPER_CONNECT=zookeeper:2181 -e KAFKA_ADVERTISED_HOST_NAME=127.0.0.1 \
    -v /var/run/docker.sock:/var/run/docker.sock \
    wurstmeister/kafka
```

File: deploy/docker-compose/docker-compose.yml [hunk 1]
```diff
@@ -14,25 +14,37 @@ services:
...
-      - '4222'
+      - '2181:2181'
+
+  kafka:
+    image: wurstmeister/kafka
+    ports:
+      - '9092:9092'
+    environment:
+      KAFKA_ADVERTISED_PORT: "9092"
+      KAFKA_ZOOKEEPER_CONNECT: "zookeeper:2181"
```
Comment (koshelev) [inline deploy/docker-compose/docker-compose.yml]: I'm not sure if we need link to zookeeper for this to work

**Comment by jerrytrieu** [comment]
@alexhumphreys can you run this on Genivi instance?

**Comment by alexhumphreys** [comment]
@jerrytrieu Can be done. You want me to run this now as a test? Or just in general when it is merged?

**Pull request merged**